src/app/photos/photos.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+import { PhotosService, Photo } from './services/photos/photos.service';
```

Comment:
The path which follows the keyword `from` can actually be relative to the current file. I suggest reading [this](https://stackoverflow.com/questions/21306512/difference-between-relative-path-and-absolute-path-in-javascript) to get a quick idea of how it works. To be specific, the path should be `../services/photos/photos.service` instead of `./services/photos/photos.service`. Of course this will later change again once you move this component inside `src/app/pages`.

Reply:
I see!! The explanation in the link provided is really clear, thx Clarence!